src/main/jbake/content/get-involved.ad

```diff
@@ -22,18 +22,47 @@
 :jbake-status: published
 :idprefix:
 
-
 The Apache OpenNLP project is developed by volunteers and is always looking for new contributors
 to work on all parts of the project. Every contribution is welcome and needed to make it better.
-A contribution can be anything from a small documentation typo fix to a new component.
+A contribution can be anything from a small documentation typo fix to a new component. We especially welcome contributions from first-time users.
+
+To get started, please first join the Apache OpenNLP mailing lists:
+
+[cols="3,3,3,3,3"]
+|===
+|Mailing List |Purpose |Subscribe |Unsubscribe |Archives
+
+|Users Mailing List
+|This list is the general users mailing list and is for posting most, if not all, relevant questions. If you are trying to decide which list, opennlp-dev or opennlp-users, post here first. Someone will kindly suggest you post to opennlp-dev if it is more appropriate there.
+|mailto:users-subscribe@users.apache.org[Subscribe]
+|mailto:users-unsubscribe@users.apache.org[Unsubscribe]
+|http://mail-archives.apache.org/mod_mbox/opennlp-users/[Archives]
```

Comment:
https://lists.apache.org/list.html?users@opennlp.apache.org ?

It's redirected there when I open it, and some browsers/companies have started blocking http links too.